pkg/playground/react-demo-cards.jsx

```diff
@@ -0,0 +1,96 @@
+ *
+ * Copyright (C) 2019 Red Hat, Inc.
+ *
+ * Cockpit is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation; either version 2.1 of the License, or
+ * (at your option) any later version.
+ *
+ * Cockpit is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "./react-demo.css";
+
+import {
+    Button, Card, CardHeader, CardBody, CardFooter,
+    CardHead, CardActions, Gallery, GalleryItem
+} from '@patternfly/react-core';
+
+const CardsDemo = () => {
+    const cards = [
+        <Card key="card0" className="flex-card" isCompact isHoverable>
+            <CardHeader>This is a compact card</CardHeader>
+            <CardBody>I'm also hoverable</CardBody>
+            <CardFooter>Compact cards have footers</CardFooter>
+        </Card>,
+        <Card key="card1" className="flex-card">
+            <CardHead>
+                <CardActions>
+                    <input type="checkbox" />
+                    <Button className="btn">click</Button>
+                </CardActions>
+                <CardHeader>This is a card header</CardHeader>
+            </CardHead>
+            <CardBody>This is a card body</CardBody>
+            <CardFooter>And here is my footer</CardFooter>
+        </Card>,
+        <Card key="card2" className="flex-card" isCompact isHoverable>
+            <CardHeader>This is a compact card</CardHeader>
+            <CardBody>I'm also hoverable</CardBody>
+            <CardFooter>Compact cards have footers</CardFooter>
+        </Card>,
+    ];
+    const galleryCards = [
+        <GalleryItem key="card3"><Card isCompact>
+            <CardBody>I'm a card in a gallery</CardBody>
+        </Card>
+        </GalleryItem>,
+        <GalleryItem key="card4">
+            <Card isCompact>
+                <CardBody>I'm a card in a gallery</CardBody>
+            </Card>
+        </GalleryItem>,
+        <GalleryItem key="card5">
+            <Card isCompact>
+                <CardBody>I'm a card in a gallery</CardBody>
+            </Card>
+        </GalleryItem>,
+        <GalleryItem key="card6">
+            <Card isCompact>
+                <CardHeader>I have a header too</CardHeader>
+                <CardBody>I'm a card in a gallery</CardBody>
+            </Card>
+        </GalleryItem>,
+        <GalleryItem key="card7">
+            <Card isCompact>
+                <CardBody>I'm a card in a gallery</CardBody>
+            </Card>
+        </GalleryItem>,
+    ];
+    return (
+        <fragment>
```

Comment:
I thought this was  spelled `<>` now; if `<fragment>` works, fine for me, I just never saw it before (just `<React.Fragment>`).

Reply:
Huh, firefox complains about <fragment>; i'll replace it with <>